src/test/unit/specific_feature_tests/undo_redo.js

```diff
@@ -358,3 +359,27 @@ test("Redo keyboard shortcut", function () {
         }
     });
 });
+
+test("Undo limit is 100", function () {
+    manipulationTestHelper({
+        startHtml: "<p>0</p>",
+        prepareFunc: function (wymeditor) {
+            wymeditor.undoRedo.reset(); // does not create an undo point
+            // create 200 undo points
+            callNTimes(function (index) {
+                wymeditor.rawHtml("<p>" + (index + 1) + "</p>");
+                wymeditor.registerModification();
+            }, 200);
+        },
+        expectedStartHtml: "<p>200</p>",
+        manipulationFunc: function (wymeditor) {
+            // try going back 101 times
+            callNTimes(function () {
+                wymeditor.undoRedo.undo();
+            }, 101);
+        },
+        // if there was no limit or the limit was higher than 100, then
+        // it would have been `<p>99</p>`
+        expectedResultHtml: "<p>100</p>"
+    });
+});
```

Comment:
Looks readable to me. I like the counting to test the limit in a very explicit way.